src/service.rs

```diff
@@ -1529,10 +1528,7 @@ impl Service {
                 let request_body = query.target().rpc_request(return_peer);
                 Poll::Ready(QueryEvent::Waiting(query.id(), node_id, request_body))
             }
-            QueryPoolState::Timeout(query) => {
-                warn!("Query id: {:?} timed out", query.id());
-                Poll::Ready(QueryEvent::TimedOut(Box::new(query)))
-            }
+            QueryPoolState::Timeout(query) => Poll::Ready(QueryEvent::TimedOut(Box::new(query))),
```

Comment:
The reason why I removed it is because a query timeout is on itself an event emited via the callback. Also queries belong to request so I'm not sure the query id on itself is useful info. 

Do you still prefer to have this as a debug statement?